driftbase/richpresence.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import annotations
+from driftbase.flask.flaskproxy import g
+from marshmallow import Schema, fields
+from marshmallow.decorators import post_load
+from driftbase.messages import post_message
+from driftbase.models.db import Friendship
+
+class PlayerRichPresence:
+    """
+    Rich presence information for a particular player.
+    @see PlayerRichPresenceSchema
+    """
+    
+    game_mode = ""
+    map_name = ""
+    is_online = False
+    is_in_game = False
```

Comment:
Do these serve any purpose?

Reply:
Can you clarify what you mean? Are you suggesting we drop the two bools from the rich presence system?

Reply:
I mean why have those class variables.

Reply:
Ah yes, you're right. Probably a leftover from using a @dataclass.